ply-plutarch/src/Ply/Plutarch/TypedWriter.hs

```diff
@@ -57,6 +63,32 @@ writeTypedScript conf descr fp target =
   where
     (ver, rl, paramTypes, scrpt) = typedWriterInfo conf target
 
+{- | Make a 'TypedScriptEnvelope' from Plutarch validator or minting policy.
+
+Unlike 'writeTypedScript', it does not write to filesystem.
+-}
+makeTypedScript ::
```

Comment:
Correct me if I'm wrong but one could already do this just with `typedWriterInfo` correct? I do appreciate the utility function though.